docs/markers.rst

```diff
@@ -16,7 +16,7 @@ Usage
     >>> marker = Marker("python_version>'2'")
     >>> marker
     <Marker('python_version > "2"')>
-    >>> # We can evaluate a marker to see if the dependency is required
```

Comment:
This directly follows the main text : `One extra requirement of dealing with dependencies is the ability to specify
if it is required depending on the operating system or Python version in use.`
I don't feel the dependency (which one ? python_version ?) is being met (but I'm not a native speaker).

To match the style of the following comments, maybe: `# Markers can be evaluated` ?

(and yes it's a long comment for a one-line change ^^)

Reply:
I simplified the sentence a bit.